recipes/mdanse/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "mdanse" %}
+{% set version = "2.0.0a1" %}
```

Comment:
Can you point this to a release?

Reply:
Unfortunately, they currently only have the prerelease published in pypi https://pypi.org/project/MDANSE/#history

Reply:
Ah, 2.0 indicated that it might not be the initial version. If it's the only one, then it's fine.